web-marketplace/src/components/organisms/EditProfileForm/EditProfileForm.tsx

```diff
@@ -65,29 +65,13 @@ const EditProfileForm: React.FC<React.PropsWithChildren<EditProfileFormProps>> =
       mode: 'onBlur',
     });
     const setErrorBannerTextAtom = useSetAtom(errorBannerTextAtom);
-
     const { isSubmitting, errors, isDirty } = useFormState({
       control: form.control,
     });
 
     /* Fields watch */
 
-    const profileType = useWatch({
```

Comment:
maybe we can just leave this file as it was initially since we don't need the custom implementation?